doc/release-notes-19202.md

```diff
@@ -0,0 +1,3 @@
+- The `-debug=db` logging category, which was deprecated in 0.20 and replaced by
```

Comment:
missing lines; need to specify category what this changes are for:

```suggestion
+Updated settings
----------------

- The `-debug=db` logging category, which was deprecated in 0.20 and replaced by
```

Reply:
Updated